### ISSUE TITLE
Get crosshair position

### ISSUE THREAD

**Issue by bzarie26** [PyMoDAQ/PyMoDAQ#491]
On daq_viewer2D, when I use the crosshair, I want to retrieve its position in order to obtain some information. However, it doesn't exist any method available which allow to get this information.

Is it possible to have a method like "get_crosshair_position" that I can use in my plugin?

Thank you in advance for any help or guidance you may be able to provide.

**Comment by loicguilmard** [comment]
Sebastien wrote an exemple in [DataPicker](https://github.com/PyMoDAQ/pymodaq_gui/blob/5.0.x_dev/src/pymodaq_gui/examples/data_picker.py)
I think it's from the viewer :
```
            xs, ys = self.viewer.view.unscale_axis(posx, posy)
            data_at = self.viewer.view.get_data_at('red', (xs, ys))
```

If you are using PyMoDAQ 4, here are the version 4 import path for the data_picker.py :

```
from qtpy.QtCore import Qt
from qtpy import QtWidgets

from typing import Optional

from pymodaq.utils.data import DataRaw, Axis


from pymodaq.utils.gui_utils.widgets.table import SpinBoxDelegate
from pymodaq.utils.parameter.utils import get_widget_from_tree
from pymodaq.utils.plotting.data_viewers.Viewer2D import Viewer2D
from pymodaq.utils.gui_util import CustomApp, Dock, DockArea

from pymodaq.utils.parameter.pymodaq_ptypes.tableview import TableViewCustom
from pymodaq.utils.scanner.scanners.tabular import TableModelTabular
```

**Comment by seb5g** [comment]
solved in #492

**Comment by seb5g** [comment]
you'll need to use version 5 from github to use it for the moment
